automation/src/main/java/org/greenplum/pxf/automation/components/gpdb/Gpdb.java

```diff
@@ -173,8 +173,11 @@ private void createForeignServers(boolean ignoreFail) throws Exception {
 		List<String> servers = Lists.newArrayList(
 		"default_hdfs",
 		"default_hive",
+		"db_hive_jdbc", //Needed for JdbcHiveTest
```

Comment:
nit here and below as well
```suggestion
		"db_hive_jdbc", // Needed for JdbcHiveTest